spring-boot-tools/spring-boot-maven-plugin/src/main/java/org/springframework/boot/maven/RunArguments.java

```diff
@@ -31,14 +32,17 @@
 
 	private static final String[] NO_ARGS = {};
 
-	private final LinkedList<String> args;
+	private final LinkedList<String> args = new LinkedList<>();
```

Comment:
it's better to initialize instance variables inside constructors instead of at the declaration level